_assets/stylesheets/pages/_app.scss

```diff
@@ -0,0 +1,220 @@
+
+
+/* iPhone X & Pixel 2 XL */
+@media (max-height: 896px) and (max-width: 414px) {
+  .jumbo-viewport {
+    min-height: 44vh;
```

Comment:
In styles there is the idea of a "magic number". It's basically when someone hardcodes a value to something that looks good. Old article here: https://css-tricks.com/magic-numbers-in-css/ on why they aren't a good thing in css.